plugins/inputs/filecount/README.md

```diff
@@ -5,11 +5,23 @@ Counts files in directories that match certain criteria.
 ### Configuration:
 
 ```toml
-# Count files in a directory
+# Count files in a directory and compute their size
 [[inputs.filecount]]
   ## Directory to gather stats about.
+  ##   deprecated in 1.9; use the directories option
   directory = "/var/cache/apt/archives"
 
+  ## Directories to gather stats about.
+  ## This accept standard unit glob matching rules, but with the addition of
+  ## ** as a "super asterisk". ie:
+  ##   /var/log/**    -> recursively find all directories in /var/log and count files in each directories
+  ##   /var/log/*/*   -> find all directories with a parent dir in /var/log and count files in each directories
+  ##   /var/log       -> count all files in /var/log and all of its subdirectories
+  directories = ["/var/cache/apt/archives"]
+
+  ## Also compute total size of matched elements. Defaults to false.
+  count_size = false
```

Comment:
This operation should be left to query time, or listing the parent directory should be functionally the same.